service/src/main/java/com/ge/predix/acs/zone/management/dao/AttributeAdapterConnectionEntity.java

```diff
@@ -0,0 +1,70 @@
+package com.ge.predix.acs.zone.management.dao;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+@Entity
+@Table(name = "attribute_adapter_connection")
+public class AttributeAdapterConnectionEntity {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "connector_id", referencedColumnName = "id", nullable = false, updatable = false)
+    private AttributeConnectorEntity connector;
+
+    @Column(name = "adapter_endpoint", nullable = false)
```

Comment:
Looks like we have not specified length for the endpoint, please check other fields as well.